Documentation/ApiOverview/Localization/TranslationServer/Crowdin/Bestpractice.rst

```diff
@@ -0,0 +1,278 @@
+
+Working with Glossary is work-in-progress, and you are welcome to maintain it in
+your preferred language. If you come across new terms or see opportunities to
+improve definitions, you can contribute directly in Crowdin. This strengthens
+collaboration and makes it easier for future translators to deliver high quality.
+
+You can read more about creating and using Glossary on Crowdin Docs: `Glossary | Crowdin Docs`_
+
+.. _Glossary | Crowdin Docs: https://support.crowdin.com/glossary/
+
+..  _crowdin-bestpractice-tm:
+
+Translation Memory (TM)
+-----------------------
+
+Translation Memory(TM) is a central part of a translation tool like Crowdin. TM
+consists of a database that constantly stores words and phrases as you go and
+offers previous translations as you work. This saves time and ensures a consistent
+translation style, especially on large projects where many strings recur across files.
+
+In TYPO3 we have a common TM – The TYPO3 CMS Great TM. Here, a large library is
+slowly and patiently built up, which helps in the daily work. TM will constantly
+suggest strings from TM that match either 100% or close by. In this way, you can
+quickly move on in the work and avoid doing duplicate work. Consider using existing
+translated strings instead of adding new ones, as this ensures consistency across
+the project.
+
+Only translated texts and words that have been approved by a proofreader are stored
+in TM. It means a lot for the quality of TM that we do not mix up too many 'almost
+identical' words and sentences in the same TM. A good TM makes it easier to achieve
+consistency in translations, and you don't have to reinvent the wheel repeatedly.
+
+Read more about Translation Memory on Crowdin Docs: `Translation Memory | Crowdin Docs`_
+
+.. _Translation Memory | Crowdin Docs: https://support.crowdin.com/translation-memory/
+
+..  _crowdin-bestpractice-shortcuts:
+
+Shortcuts and efficiency
+------------------------
+
+In Editor View, you can access a wide range of shortcuts via the keyboard. You
+can view the list by clicking on the small keyboard icon in the top right corner.
+Shortcuts can save you a lot of clicks and make it easier to navigate, especially
+when you're working on large projects. Check out the list of shortcuts and try it
+out – you'll quickly discover which ones best suit your working style.
+
+.. _crowdin-bestpractice-communication:
+
+Communication and collaboration
+-------------------------------
+
+Write, comment and suggest! In TYPO3, we primarily have two channels in Slack:
+TYPO3 translations for questions and problems about working with translations,
+and typo3 localization team for driving the process around Crowdin, TYPO3 and
+localization. It is important to use these channels actively, as collaboration
+and knowledge sharing often lead to better and more accurate translations. You
+can ask questions, share experiences, and help others if you encounter challenges.
+
+Go to TYPO3 Slack:
```

Comment:
Please link TYPO3 Slack to https://docs.typo3.org/m/typo3/guide-step-by-step/main/en-us/10GettingStarted/05MeetTheCommunity/SignUpForATypo3SlackAccount.html